src/components/ResultList/index.tsx

```diff
@@ -64,42 +62,19 @@ export const ResultList = (props: Props) => {
             <TableBody>
               {items
                 .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
-                .map((item: Item, index) => {
+                .map((item: Item) => {
                   const name = item.name._attributes.value
                   const year = item.yearpublished
                     ? item.yearpublished._attributes.value
                     : "N/A"
                   const bgGeekID = item._attributes.id
                   return (
-                    <TableRow key={index}>
-                      <TableCell component="th" scope="row">
-                        {name}
-                      </TableCell>
-                      <TableCell align="right">{year}</TableCell>
-                      <TableCell align="right">
-                        <Button
-                          onClick={() => {
-                            console.log("Button was clicked!", bgGeekID)
-                            myAPI
-                              .post("/auth/addgame", {
-                                userId: localStorage.userId,
-                                bgGeekID,
-                              })
-                              .then(res => {
-                                console.log(res)
-                                //setAdded(true)
-                                setAdded(true)
-                              })
-                              .catch(err => {
-                                console.log(err)
-                              })
-                          }}
-                          disabled={added}
-                        >
-                          Add
-                        </Button>
-                      </TableCell>
-                    </TableRow>
+                    <ResultListRow
```

Comment:
I like how you pulled all of this out in a separate component and used it here 👍 so much cleaner!